syz-ci/syz-ci.go

```diff
@@ -256,6 +261,7 @@ func main() {
 		}
 	})
 
+	go deprecateAssets(cfg, shutdownPending)
```

Comment:
I think it's better to also do wg.Add/Done for deprecateAssets to not abort it in the middle.